test/e2e/playwright/consolidation-orders.spec.ts

```diff
@@ -14,7 +14,7 @@ test.describe('Consolidation Orders', () => {
       predicate: (e) => e.url().includes('api/offices'),
     });
 
-    await page.goto('/data-verification');
+    await page.goto('/data-verification?x-ms-routing-name=staging');
```

Comment:
The following query string is needed to target the slot environment. This should be refactor to use fixtures so that this doesn't need to be hard coded everywhere.

See https://playwright.dev/docs/test-fixtures#creating-a-fixture

Reply:
Fixture added to add x-ms-routing-name